borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,88 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The itemid to publish to
+                    for details.
+                example: 55105
+            host:
+                type: string
+                description: |
+                    Host name where the item is stored.
+                    Required if 'itemid' is not set
+                example: borg-server
+            key:
+                type: string
+                description: |
+                    Key of the host where the item is stored.
+                    Required if 'itemid' is not set
```

Comment:
A host can have multiple items associated with them. Each item can be uniquely identified across the entire system with an `itemid`. Each items has a name as well as a human readable `key`. The keys do not need to be unqiue across the system. For example:

**Host Name:** "Borg Server Head Office"
**Item Visible Name:** Borg Status Domain Controller
**Item ID:** 50010
**Item Key:** borg.status.domain
**Item Value:** 0

**Host Name:** "Borg Server Datacenter"
**Item Visible Name:** Borg Status Domain Controller
**Item ID:** 59919
**Item Key:** borg.status.domain
**Item Value:** 0

Notice how both hosts can have the same `key` for each host but they cannot have the same `itemid`.

This is why in the API, you can either publish your values to an `itemid` or a pair of `host` and `key`

Here is the commit to fix the quotes and period: https://github.com/borgmatic-collective/borgmatic/pull/85/commits/702e55e6f7976fd7e2acaa383c526f386ff84757

Reply:
Ah okay, thanks for explaining! So maybe something like this would work: "Zabbix key where the monitoring data is written. Required along with host if itemid is not set."